libmogwai-schedule/schedule-service.c

```diff
@@ -536,7 +553,7 @@ entries_changed_cb (MwsScheduler *scheduler,
   if (((added == NULL) != (removed == NULL)) ||
       (added != NULL && removed != NULL && added->len != removed->len))
     {
-      notify_scheduler_properties (self);
+      notify_scheduler_properties (self, TRUE, FALSE);
```

Comment:
If we grow this in number of options to be notified, then it's going to get really confusing. So I'd suggest using a flag here `MWSC_NOTIFY_ENTRIES & MWSC_NOTIFY_ALLOW_DOWNLOADS & ...`, but we can change that only when it becomes necessary if you don't want to do it now.

Reply:
Yeah, I was thinking that when I wrote it. :+1: for changing to a set of flags if needed in future.